torch/xpu/__init__.py

```diff
@@ -66,9 +66,14 @@ def is_available() -> bool:
     return device_count() > 0
 
 
-def is_bf16_supported():
+def is_bf16_supported(including_emulation: bool = True):
```

Comment:
API align to other backend, refer to https://github.com/pytorch/pytorch/blob/7cae7902a2205ac2c9b02ad719718a1ac590c59a/torch/cuda/__init__.py#L184

Reply:
@guangyey , why does XPU need to provide this optional parameter? Is there any case where one Intel GPU does not support BF16 natively while the Intel GPU software stack emulates BF16 for the Intel GPU?

Reply:
In simple terms, eager mode depends on emulation, while graph mode requires native support to fully leverage performance optimization.

Reply:
https://github.com/intel/llvm/blob/f958dce28053dff145cd725ff57bc4ce94cb94d7/sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc#overview According to the note, the non-XE GPU may not support BF16 natively.